### PR TITLE
feat: add enrich token method

### PR DESCRIPTION
**Pull request by bakkelim**
# Checklist

- [ ] `make test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows [commit guidelines](./doc/guides/commit-message.md#commit-message-guidelines)

## Description of change

added new method ```enrich_token()``` which implements ```jarvis_sdk.indykite.identity.v1beta1.EnrichToken``` method

### CODE REVIEW

File: jarvis_sdk/cmd/_enrich_token.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from jarvis_sdk.indykite.identity.v1beta1 import identity_management_api_pb2 as pb2
+from google.protobuf import struct_pb2
+
+def enrich_token(self, user_token: str, token_claims: dict, session_claims: dict):
+    try:
+        token_struct = struct_pb2.Struct()
+        if token_claims is not None:
+            token_struct.update(token_claims)
```
Comment (arxeiss) [inline jarvis_sdk/cmd/_enrich_token.py:8]: Is this method you were searching for? The one that converts native Python type to PB2 Stuct?

**Review by arxeiss — APPROVED** [review]

File: jarvis_sdk/api.py [hunk 2]
```diff
@@ -213,6 +217,17 @@ def main():
         if dt is not None:
             print_response(dt)
 
+    elif command == "enrich-token":
+        user_token = args.user_token
+        token_claims = get_struct_test_claims()
+        session_claims = get_struct_test_claims()
```
Comment (dusan-kovacik-profiq) [inline jarvis_sdk/api.py:223]: Why are you using test claims here?

File: jarvis_sdk/cmd/_enrich_token.py (hunk 1 above)
Comment (bakkelim) [reply jarvis_sdk/cmd/_enrich_token.py:8]: yes 👍

File: jarvis_sdk/api.py (hunk 2 above)
Comment (bakkelim) [reply jarvis_sdk/api.py:223]: I assumed that api.py was used for testing purpose and could use some test data. Would it make sense to pass it through the command line?

Comment (dusan-kovacik-profiq) [reply jarvis_sdk/api.py:223]: Well, I think the api is used for running commands from CLI and everyone can use it, but that's just my understanding.

Comment (bakkelim) [reply jarvis_sdk/api.py:223]: okay, I will update this

**Review by dusan-kovacik-profiq — APPROVED** [review]

**Pull request merged**